### PR TITLE
add fallback icon for missing nft

### PR DESCRIPTION
**Pull request by diegodelrieu**
### Issue / feature description

`Adding a fallback icon where it's missing`

### CODE REVIEW

File: packages/extension/src/ui/features/accountNfts/CollectionNfts.tsx [hunk 1]
```diff
@@ -73,15 +74,19 @@ export const CollectionNfts: FC = () => {
...
-              }
-              borderRadius="lg"
-            />
+            {collectible.imageUri ? (
+              <Image
+                w="64px"
+                h="64px"
+                src={collectible.imageUri ?? undefined}
+                backgroundColor={
+                  !collectible.imageUri ? "neutrals.300" : "transparent"
```
Comment (simonheys) [inline packages/extension/src/ui/features/accountNfts/CollectionNfts.tsx:83]: There is already a check for this on line 77 - redundant? `!collectible.imageUri`

File: packages/extension/src/ui/features/accountNfts/CollectionNfts.tsx [hunk 2]
```diff
@@ -73,15 +74,19 @@ export const CollectionNfts: FC = () => {
...
-              backgroundColor={
-                !collectible.imageUri ? "neutrals.300" : "transparent"
-              }
-              borderRadius="lg"
-            />
+            {collectible.imageUri ? (
+              <Image
+                w="64px"
+                h="64px"
+                src={collectible.imageUri ?? undefined}
```
Comment (simonheys) [inline packages/extension/src/ui/features/accountNfts/CollectionNfts.tsx:81]: Same comment as below

Comment (diegodelrieu) [reply packages/extension/src/ui/features/accountNfts/CollectionNfts.tsx]: good catch ty

**Review by dhruvkelawala — APPROVED** [review]

File: packages/extension/src/ui/features/accountNfts/CollectionNfts.tsx [hunk 3]
```diff
@@ -73,15 +74,17 @@ export const CollectionNfts: FC = () => {
...
-              src={collectible.imageUri ?? undefined}
-              backgroundColor={
-                !collectible.imageUri ? "neutrals.300" : "transparent"
-              }
-              borderRadius="lg"
-            />
+            {collectible.imageUri ? (
+              <Image
+                w="64px"
+                h="64px"
```
Comment (simonheys) [inline packages/extension/src/ui/features/accountNfts/CollectionNfts.tsx:80]: Can please use spacer values instead of `px`?

**Review by simonheys — APPROVED** [review]

**Pull request merged**